.github/workflows/diff.yaml

```diff
@@ -1,8 +1,8 @@
-name: Diff changes to dist
+name: Diff changes to release
```

Comment:
I think I'm missing some context for the rationale behind renaming `dist` to `release` (and `package` to `dist`). I'm not immediately sure of the difference between distribution and release just from naming (I've seen differing uses of those words, but maybe opinion is solidifying around how we use 'em?).

Not against it at all (it links up to our "release" archives at the very least), just think we need some explanation of it in the PR's description/commit messages.

Reply:
Caught up with @romaricpascal and @36degrees today

Since the useful work is moving `govuk-frontend` we can leave `./dist` at the project root for now